packages/server/graphql/public/types/NotifyDiscussionMentioned.ts

```diff
@@ -7,7 +7,10 @@ const NotifyDiscussionMentioned: NotifyDiscussionMentionedResolvers = {
     const meeting = await dataLoader.get('newMeetings').load(meetingId)
     return meeting as MeetingTeamPrompt
   },
-  author: ({authorId}, _args: unknown, {dataLoader}) => {
+  author: async ({authorId, commentId}, _args: unknown, {dataLoader}) => {
+    const comment = await dataLoader.get('comments').load(commentId)
+    if (comment.isAnonymous) return null
```

Comment:
-1 This is a good quick fix, but we should not store the `author` in the first place for anonymous comment notifications